scripts/v120_flow.py

```diff
@@ -10,26 +11,26 @@
 
 # Wrapping the function to ryven nodes
 @Node(viewer)
-def load_bunny() -> Mesh:
-    return Mesh.from_ply(compas.get('bunny.ply'))
+def load_mesh(path: str) -> Mesh:
+    return Mesh.from_ply(compas.get(path))
```

Comment:
we should add a `from_file` method that just checks the extension and then dispatches to the more specific methods...

Reply:
> LGTM. just don't forget about the docs. otherwise they never get filled in ... :)

haha sure doing that now